tests/src/Functional/OgSelectionWidgetAutoCompleteTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Drupal\Tests\og\Functional;
+
+use Behat\Mink\Exception\ResponseTextException;
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\Og;
+use Drupal\og\OgGroupAudienceHelper;
+use Drupal\simpletest\ContentTypeCreationTrait;
```

Comment:
Deprecated, see https://www.drupal.org/node/2884454
```suggestion
use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
```